app/javascript/react/textual_summary/generic_group.jsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import GenericTableRow from './generic_table_row';
+
+export default class GenericGroup extends React.Component {
```

Comment:
> this can probably be written as a const,

yes, the ones where there'll be just the `render` and no other code I'll convert to functions. (I guess you mean "use function rather thatn class").

> and probably consider using patternfly-react table component.

not in the repo at this point, but yes, I will take a look if it's usable in this context